src/main/clj/parti_time/output/api.clj

```diff
@@ -1,15 +1,12 @@
 (ns parti-time.output.api)
 
 (defn file-extension [filename timeline]
-  (re-find #"(?<=\.)[^.]+$" filename))
+  (or (re-find #"(?<=\.)[^.]+$" filename)
+      "tl"))
 
 (defmulti write-timeline
   file-extension)
 
-(defmethod write-timeline nil [filename timeline]
```

Comment:
The default dispatch is not needed any more, since our dispatch function fills in the default value "tl" for nil.